### DISCUSSION TITLE
Don't stick Icon and file name together

### DISCUSSION THREAD

**Discussion by 1aron** [community/community#39807] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Leave a space of 0.5rem between the icon and the file name.
<img width="1178" alt="Screen Shot 2022-11-23 at 2 17 39 PM" src="https://user-images.githubusercontent.com/33840671/203483418-8b06ce25-8327-4a27-9564-4d96cc6f621b.png">

**Comment by dennybill** [comment]
People usually keep the icon element as an empty node and put the text content after it. Then you add a margin to the icon to get appropriate spacing:

```
.facebook .fab { margin-right: 1em }

<a href="..." class="facebook">
   <i class="fab fa-facebook"></i>
   <span>Facebook</span>
</a>
```
The <span> tags are technically not necessary, but it is common to avoid [text](https://gbwa.download/)-only nodes.
